### PR TITLE
chore(deploy): update release scripts to not use 3 different images

### PR DESCRIPTION
**Pull request by ecordell**
This is to fix: https://gitlab.com/operator-framework/operator-framework_operator-lifecycle-manager/-/jobs/106854881

### CODE REVIEW

File: .gitlab-ci/base_jobs.libsonnet [hunk 1]
```diff
@@ -22,16 +22,16 @@ local appr = utils.appr;
 
     AlmTest: {
```
Comment (njhale) [inline .gitlab-ci/base_jobs.libsonnet]: OlmTest?

Comment (ecordell) [reply .gitlab-ci/base_jobs.libsonnet]: Deleted - I removed the unit tests from gitlab since they run in the other ci system now

**Pull request merged**